### DISCUSSION TITLE
Which model architecture do you suggest to use for Turkish dataset?

### DISCUSSION THREAD

**Discussion by burhancetinkaya** [coqui-ai/TTS#2732] [General Q&A]
Hi everyone,
I have a Turkish speech dataset with transcripts generated (about 100gb) using Whisper large-v2. All the voices are clean and professionally recorded. And also divided into folders by gender. Each gender folder has multiple speaker voices.

So, I want to train a model for Turkish TTS (Text-to-Speech). However, I am unsure how to decide which model architecture to choose. Do you have any training experience that could help me decide which one to use?
Thanks in advance

cc: @erogol

**Comment by erogol** [comment]
You can use any. Let me know if you are willing to open-source the model.

**Reply by burhancetinkaya** [reply]
Hi Eren, My team have been working on it since July. We have decided to use vits backbone. Results are so impressive. We also use vits for voice cloning. Because of limited gpu resource, It is still in training. We build this for Lena product for enterprise companies in Turkey. I am willing to open-source the model when our product goes live.
